ledger/participant-integration-api/src/main/scala/platform/configuration/IndexConfiguration.scala

```diff
@@ -7,4 +7,6 @@ object IndexConfiguration {
 
   val DefaultEventsPageSize: Int = 1000
 
+  val DefaultConcurrency: Int = 16
```

Comment:
What is the magic behind this specific number?

Reply:
It's more than one. :stuck_out_tongue:

Reply:
16 seems to have many properties, the closest to magic I can find seems to be this: https://en.wikipedia.org/wiki/16_(number)#Hinduism

:trollface: